vetur/tags.json

```
@@ -1,10 +1,10 @@
 {
   "router-view": {
     "attributes": ["name", "route"],
-    "description": "Component that renders the matched component for the current location. Components rendered by `<router-view>` can also contain their own `<router-view>` to render nested routes."
+    "description": "Component that renders the matched component for the current location. Components rendered by `<router-view>` can also contain their own `<router-view>` to render nested routes.\n\n[API Reference](https://router.vuejs.org/api/#router-link)"
   },
   "router-link": {
     "attributes": ["to", "activeClass","exactActiveClass", "custom"],
-    "description": "Component that renders an `<a>` with the correct `href` attribute and click listeners to trigger a local navigation when clicked. Can also customize its rendering by providing the `custom` prop and using its `v-slot` API."
+    "description": "Component that renders an `<a>` with the correct `href` attribute and click listeners to trigger a local navigation when clicked. Can also customize its rendering by providing the `custom` prop and using its `v-slot` API.\n\n[API Reference](https://router.vuejs.org/api/##router-view)"
```

there is a double `#`

```suggestion
    "description": "Component that renders an `<a>` with the correct `href` attribute and click listeners to trigger a local navigation when clicked. Can also customize its rendering by providing the `custom` prop and using its `v-slot` API.\n\n[API Reference](https://router.vuejs.org/api/#router-view)"
```